pumpkin/src/command/commands/cmd_pumpkin.rs

```diff
@@ -35,5 +38,7 @@ impl CommandExecutor for PumpkinExecutor {
 }
 
 pub fn init_command_tree<'a>() -> CommandTree<'a> {
-    CommandTree::new(NAMES, DESCRIPTION).execute(&PumpkinExecutor)
+    CommandTree::new(NAMES, DESCRIPTION)
+        .with_child(literal("test").with_child(pumpkin_test_client_side_arg_parsers()))
```

Comment:
I really don't like this.
1. "test" does really say nothing what is being tested.
2. I think just rust unit testing would be enough

Reply:
it's not just test, there are more literal arguments that are more specific. What this is for can't be done in a unit test because it's about seeing how the client handles each argument type in game. This way I could check whether I got all the argument type properies correct for example. I think I am going to remove it for now tho because it was just a really low effort way of doing this